runtime/plugin/matchit.vim

```diff
@@ -1,8 +1,10 @@
 "  matchit.vim: (global plugin) Extended "%" matching
-"  Last Change: Fri Jul 29 01:20 AM 2016 EST
+"  Last Change: 2016 Aug 21
 "  Maintainer:  Benji Fisher PhD   <benji@member.AMS.org>
 "  Version:     1.13.2, for Vim 6.3+
 "		Fix from Tommy Allen included.
```

Comment:
thanks for those nice words. I am just trying to be helpful ;)
I just wondered the same, so forwarded that mail directly to his address. Perhaps he'll see it.